common/utils.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"context"
+
+	"cloud.google.com/go/spanner"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type NewClientConfig struct {
+	DatabaseName string
+	Endpoint     string
+}
+
+func NewClient(ctx context.Context, config NewClientConfig) (*spanner.Client, error) {
+	options := []option.ClientOption{
+		option.WithGRPCDialOption(grpc.WithTransportCredentials(insecure.NewCredentials())),
+		option.WithoutAuthentication(),
+	}
+	if config.Endpoint != "" {
```

Comment:
Are we setting a default endpoint somewhere? If not, I would make this one required and let the validations do the job.

Reply:
It works a bit like the [kinesis connector](https://github.com/conduitio-labs/conduit-connector-kinesis/blob/194ca3afc288a196213917b42e1c54c19dedf688/common/client.go#L49-L58). By default, the google spanner client hits google servers. Having an optional `config.Endpoint` allows me to hit the local docker spanner emulator.

Now, I still haven't end2end tested that my google spanner client usage properly connects to the production servers. I guess it shouldn't, as I'm not using any credentials.

There's an issue already for this #14.